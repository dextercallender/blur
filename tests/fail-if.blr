int main() {
    if('h') {
        println("if true test");
    }
}
