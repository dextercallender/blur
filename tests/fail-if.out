Fatal error: exception Failure("illogical if")
